packages/ecs-morgan-format/index.js

```diff
@@ -175,6 +175,8 @@ function ecsFormat (opts) {
     formatHttpRequest(ecsFields, req)
     formatHttpResponse(ecsFields, res)
 
+    opts.customize && opts.customize(ecsFields, req, res);
```

Comment:
How about changing this to:

1. an option called `logHook`. Here I'm leaning a bit towards a *similar* option for configuring OpenTelemetry instrumentations for some logging frameworks. E.g. this one for Bunyan: https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-bunyan/#bunyan-instrumentation-options
2. Change the `logHook` to take an object with named params:

```suggestion
    opts.logHook && opts.logHook({record: ecsFields, req, res});
```

I'm using "record" to partly mimic that same OpenTelemetry example. However, using an object here does differ. I'm leaning towards an object over positional args because if this were to be added to other logging frameworks (like winston, pino, etc.), the association with HTTP requests and response objects isn't as much of a given as it is for morgan -- which is all about HTTP logging.

I don't know if having an object arg is a pain for the .d.ts file.

Reply:
Thanks. Updated the signature to your suggestion.